src/GitHub.VisualStudio/GitHub.VisualStudio.csproj

```diff
@@ -414,6 +414,10 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
       <IncludeInVSIX>true</IncludeInVSIX>
     </Content>
+    <Content Include="System.ValueTuple.dll">
```

Comment:
Hm, this approach of adding the assembly to the repo seems a little odd. Why not just add the nuget package with the proper version?

Reply:
I find the rules for when a file is actually included in the .vsix to be super confusing.

It seems like all of the following are required:
1. Appear as Asset in .vsixmanifest
2. Appear as Content in .csproj
3. IncludeInVSIX=true
4. CopyToOutputDirectory

I hoped I'd be able to simply add Assets to .vsixmanifest that point to files in the output directory. Alas this doesn't seems to work (the extra ceremony is required). 😕 

I've updated the project to using a Link rather than including the binary. This is maybe a little better, but still not great. Let me know if you find a cleaner way!

Reply:
Ah, ok. I thought adding the NuGet package would automatically include the assembly from it in the VSIX.

Reply:
That's what I hoped as well, bit it looks like there are differences between a plain old assembly reference and a `<PackageReference>`. I ended up with missing assemblies when I converted TestDriven.Net to use `<PackageReference>` as well. 😢